src/shared/components/organisms/navbar/Navbar.tsx

```diff
@@ -44,8 +45,14 @@ const Navbar: React.FC<INavbar> = ({ isUserLoggedin, navbarItemList }) => {
   };
 
   const getUserEmail = () => {
-    const { userDetails } = globalAuthData;
-    return userDetails.email;
+    const jwt = localStorage.getItem('jwt_token') || '';
+    try {
+      const { username }: { username: string } = JWTDecode(jwt);
```

Comment:
A better approach would be to send the username along with the token from backend when requesting for a JWT with a google auth token